sebs/aws/aws.py

```
@@ -368,23 +355,49 @@ def update_function(
-            FunctionName=name,
-            Timeout=function.config.timeout,
-            MemorySize=function.config.memory,
-        )
-        self.wait_function_updated(function)
-        self.logging.info(f"Updated configuration of {name} function. ")
-        self.wait_function_updated(function)
-        self.logging.info("Published new function code")
+        self.update_function_configuration(function, code_package)
+
+    def update_function_configuration(
+        self, function: Function, code_package: Benchmark, env_variables: dict = {}
```

Ruff (0.8.2)

Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)